src/TreeView/TreeView.tsx

```diff
@@ -304,6 +329,7 @@ const Item: React.FC<TreeViewItemProps> = ({defaultExpanded = false, onSelect, o
             }
           }}
           sx={{
```

Comment:
The value of `sx` could get reallly big as we start to get more into the styling. Maybe we could consider moving the style code into it's own file or at least outside of the component function's scope.

(Non-blocking for this PR)